blockchain/runtime/src/lib.rs

```diff
@@ -266,9 +266,9 @@ impl pallet_sudo::Config for Runtime {
 }
 
 parameter_types! {
-    pub const MaxRewardPerUser: Balance = 1 << 60;
-    pub const MaxMintPerPeriod: Balance = 1 << 64;
-    pub const MintEveryNBlocks: BlockNumber = 10;
+    pub const MaxRewardPerUser: Balance = 1.5e18 as Balance;
```

Comment:
Does rust guarantee that `1.5e18 as u128` actually produces 1,500,000,000,000,000,000? It seems to do that from my tests, but I'm not sure about guarantees...

I think I'd prefer something like `1_500 * MILLI_UNIT` for this and `80_000 * UNIT` for max mint.